docs/develop/automatically-open-on-installation.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Automatically open a task pane when an add-in is installed
+description: Learn how to configure an Office Add-in to open automatically when it's installed.
+ms.date: 09/01/2022
+ms.localizationpriority: medium
+---
+
+
+# Automatically open a task pane when an add-in is installed
+
+You can configure your add-in's task pane to launch immediately after it's installed. This feature increases usage. 
+
+By default, when a task pane add-in that does *not* include any [add-in commands](../design/add-in-commands.md) is installed, it's task pane opens immediately upon installation. But when the add-in does have one or more add-in commands, the add-in's ribbon tab is selected and a callout appears near it notifying the user of the new add-in, but the add-in doesn't launch until the user takes action to start it, such as selecting a button to open a task pane. This historic default behavior is changing so that automatic launching upon installation is restored in some situations to add-ins that include add-in commands. In addition, if the add-in has more than one task pane page, it's possible for you to control whether the add-in launches upon installation and, if so, which page opens in the task pane. 
```

Comment:
Removing apostrophe from possessive "its". 

```suggestion
By default, when a task pane add-in that does *not* include any [add-in commands](../design/add-in-commands.md) is installed, its task pane opens immediately upon installation. But when the add-in does have one or more add-in commands, the add-in's ribbon tab is selected and a callout appears near it notifying the user of the new add-in, but the add-in doesn't launch until the user takes action to start it, such as selecting a button to open a task pane. This historic default behavior is changing so that automatic launching upon installation is restored in some situations to add-ins that include add-in commands. In addition, if the add-in has more than one task pane page, it's possible for you to control whether the add-in launches upon installation and, if so, which page opens in the task pane. 
```